### PR TITLE
[release-4.14] Fix pinned kernel and CRI-O version

### PR DESCRIPTION
**Pull request by vrutkovs**
Fixes [#1798](https://github.com/okd-project/okd/issues/1798)

### CODE REVIEW

**Comment by titou10titou10** [comment]
@vrutkovs in Dockerfile, you replaced FCOS version by `6.5.4-200.fc38` where the "bad" version was`6.5.4-300.fc39` (`v4.14.0-0.okd-2023-11-12-042703`)

but...

the current kernel version of OKD is`6.5.5-200.fc38` (ie for`v4.14.0-0.okd-2023-10-28-073550`)

Q: is that intentional?

File: Dockerfile [hunk 1]
```diff
@@ -26,10 +26,10 @@ RUN cat /etc/os-release \
         /tmp/rpms/$([ -d /tmp/rpms/$(uname -m) ] && echo $(uname -m)/)openshift-hyperkube-*.rpm \
     && rpm-ostree cliwrap install-to-root / \
     && rpm-ostree override replace \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-6.5.4-300.fc39.x86_64.rpm \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-core-6.5.4-300.fc39.x86_64.rpm \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-modules-6.5.4-300.fc39.x86_64.rpm \
-    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/300.fc39/x86_64/kernel-modules-core-6.5.4-300.fc39.x86_64.rpm \
+    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.4/200.fc38/x86_64/kernel-6.5.4-200.fc38.x86_64.rpm \
```
Comment (LorbusChris) [inline Dockerfile:29]:
```suggestion
    https://kojipkgs.fedoraproject.org//packages/kernel/6.5.5/200.fc38/x86_64/kernel-6.5.5-200.fc38.x86_64.rpm \
```

Comment (vrutkovs) [reply Dockerfile:29]: We want 6.5.4 to match the first 4.14 stable

**Comment by vrutkovs** [comment]
>the current kernel version of OKD is6.5.5-200.fc38 (ie forv4.14.0-0.okd-2023-10-28-073550)

That's not what I'm seeing. [nodes](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_okd-machine-os/690/pull-ci-openshift-okd-machine-os-release-4.14-e2e-aws-ovn/1724161153694699520) show that they are running 6.5.4 in [4.14.0-0.okd-2023-11-13-192150](https://amd64.origin.releases.ci.openshift.org/releasestream/4.14.0-0.okd/release/4.14.0-0.okd-2023-11-13-192150)

**Comment by LorbusChris** [comment]
@vrutkovs apologies, I only commented on the first occurrence of the version, but there are more..

**Comment by llomgui** [comment]
@vrutkovs @LorbusChris You do not need to specify the kernel. It will pick the correct one.
I'm currently testing the image and it works. Kernel in used: 6.5.8-200.fc38.x86_64

**Comment by vrutkovs** [comment]
>I'm currently testing the image and it works

NFS tests fail with kernels .6-.8, see https://github.com/coreos/fedora-coreos-tracker/issues/1601

**Comment by llomgui** [comment]
Ah, it's a master, I didn't test NFS stuff.

**Pull request merged**